### PR TITLE
Adds Endpoint.ipv6 to thrift and openapi spec

### PR DESCRIPTION
**Pull request by adriancole**
See #306 for details

### CODE REVIEW

File: zipkin-api.yaml [hunk 1]
```diff
@@ -201,8 +201,13 @@ definitions:
     properties:
       serviceName:
         type: string
+        description: 'Classifier of this endpoint in lowercase, such as "acme-frontend"'
       ipv4:
-        type: string
+        type: ipv4
```
Comment (adriancole) [inline zipkin-api.yaml]: @mohsen1 json schema types like this are legit, right?

Comment (mohsen1) [inline zipkin-api.yaml]: No, `ipv4` is a format. JSON Schema types are only these:

```
array
A JSON array.
boolean
A JSON boolean.
integer
A JSON number without a fraction or exponent part.
number
Any JSON number. Number includes integer.
null
The JSON null value.
object
A JSON object.
string
A JSON string.
```

Comment (adriancole) [inline zipkin-api.yaml]: thanks.. surprised travis didn't crash!

Comment (adriancole) [inline zipkin-api.yaml]: fixed

**Pull request merged**